lib/Common/PlatformAgnostic/Exception.h

```diff
@@ -0,0 +1,15 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+#ifdef __clang__
+    #undef __try
+    #undef __finally
+    #define __try
+    #define __finally
```

Comment:
This looks overkill to me and brings more problems than good. We discourage SEH use because that's not cross platform. Better not to have them available than to define them.